packages/tags/src/Tag.ts

```diff
@@ -90,11 +90,16 @@ export class Tag extends SizedMixin(SpectrumElement, {
         if (this.readonly) {
             return;
         }
-        this.dispatchEvent(
-            new Event('delete', {
-                bubbles: true,
-            })
-        );
+        const deleteEvent = new Event('delete', {
+            bubbles: true,
+            composed: true,
+        });
+        this.dispatchEvent(deleteEvent);
+
+        if (deleteEvent.defaultPrevented) {
+            return;
+        }
+        this.remove();
```

Comment:
```suggestion
        const applyDefault = this.dispatchEvent(new Event('delete', {
            bubbles: true,
            composed: true,
        }));

        if (!applyDefault) {
            return;
        }
        this.remove();
```
This normalizes this pattern to other usage in the library.